soccer/src/soccer/planning/planner/line_kick_path_planner.cpp

```diff
@@ -67,11 +67,11 @@ Trajectory LineKickPathPlanner::initial(const PlanRequest& plan_request) {
 
     LinearMotionInstant target{ball_position - offset_from_ball};
 
-    MotionCommand modified_command{"path_target", target,
+    MotionCommand modified_command{"initial_path_target", target,
                                    FacePoint{plan_request.motion_command.target.position}};
     modified_request.motion_command = modified_command;
 
-    return path_target_.plan(modified_request);
+    return initial_planner.plan(modified_request);
```

Comment:
looks like you missed the underscore after the variable name so it isn't compiling